docs/features/software-catalog/descriptor-format.md

```diff
@@ -44,7 +44,7 @@ metadata:
   name: artist-web
   description: The place to be, for great artists
   labels:
-    system: public-websites
+    backstage.io/custom: ValueStuff
```

Comment:
Let's use `example.com/custom` to make it more clear that you can customize that part of the label as well

Reply:
@Rugvip addressed! thanks for the feedback